doc/man7/provider-base.pod

```diff
@@ -104,7 +104,7 @@ provider):
  core_gettable_params           OSSL_FUNC_CORE_GETTABLE_PARAMS
  core_get_params                OSSL_FUNC_CORE_GET_PARAMS
  core_thread_start              OSSL_FUNC_CORE_THREAD_START
- core_get_library_context       OSSL_FUNC_CORE_GET_LIBRARY_CONTEXT
+ core_get_libctx       OSSL_FUNC_CORE_GET_LIBCTX
```

Comment:
Indentation here is screwed

Reply:
Addressed with c089c4c8ce3b139dd1a5e543f563e9ba8b280e4c